homeassistant/package_constraints.txt

```diff
@@ -8,7 +8,7 @@ jinja2>=2.10
 PyJWT==1.7.1
 cryptography==2.6.1
 pip>=8.0.3
-python-slugify==1.2.6
+python-slugify>=3.0.2
```

Comment:
All versions need to be pinned.

Reply:
Hm... I don't mind doing that, but certifi neither pytz, pyyaml, pycryptodome are pinned.